Form-Controls/styles.css

```diff
@@ -0,0 +1,133 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: 16px;
+}
+
+header {
+  background-color: #d5f0fd;
+  width: 90%;
+  text-align: center;
+  border-bottom: solid 5px #d8cefe;
+}
+
+main {
+  width: 90%;
+  margin: auto;
+  background-color: #ddebf8;
```

Comment:
For colors, you can use css var to define these colors at the beginning for readability. Especially, when you are reusing them multiple times later in the file. Hint: using ':root'
**Improvement 1:** define and use css var for colors